README.md

```diff
@@ -9,7 +9,7 @@ Shivam Raval
 
 
 ## Badge travisci
-[![Build Status](https://app.travis-ci.com/cs107-i-m-i-m/cs107-FinalProject.svg?token=RKxQRxaJdxdtKXuUyQLW&branch=main)](https://app.travis-ci.com/cs107-i-m-i-m/cs107-FinalProject)
+[![Build Status](https://app.travis-ci.com/cs107-i-m-i-m/cs107-FinalProject.svg?token=RKxQRxaJdxdtKXuUyQLW&branch=xinrong-dev)](https://app.travis-ci.com/cs107-i-m-i-m/cs107-FinalProject)
```

Comment:
It's a good thing to switch to your branch for testing, but I think it should be better to have it point on the main branch in the final version

Reply:
Very good point! It's changed back now.